src/db.rs

```diff
@@ -15,3 +17,26 @@ pub async fn get_userdata(client: &Client, token: &str) -> Result<User, Error> {
 
     User::from_row_ref(&queried_data)
 }
+
+pub async fn create_user(
+    client: &Client,
+    username: &str,
+    token: Vec<u8>,
```

Comment:
No idea if this will work because i couldn't find any documentation for Client but it's worth a try
```suggestion
    token: &[u8],
```